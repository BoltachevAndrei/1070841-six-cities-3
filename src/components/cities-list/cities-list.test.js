import React from 'react';
import renderer from 'react-test-renderer';
import CitiesList from './cities-list.jsx';

const offers = [
  {
    id: 111,
    city: `Test city`,
    isPremium: true,
    images: [
      `img/amsterdam.jpg`
    ],
    price: 222,
    isBookmarked: true,
    rating: 1,
    title: `Test title 1`,
    features: {
      entire: `Test type 1`,
    }
  },
  {
    id: 333,
    city: `Test city`,
    isPremium: false,
    images: [
      `img/amsterdam@2x.jpg`
    ],
    price: 444,
    isBookmarked: false,
    rating: 5,
    title: `Test title 2`,
    features: {
      entire: `Test type 2`,
    }
  }
];

const activeCity = `Test city`;

it(`CitiesList renders correctly`, () => {
  const tree = renderer
    .create(<CitiesList
      offers={offers}
      activeCity={activeCity}
      onCityClick={() => {}}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
