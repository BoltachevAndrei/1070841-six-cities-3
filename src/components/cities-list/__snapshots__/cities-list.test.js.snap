// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CitiesList renders correctly 1`] = `
<ul
  className="locations__list tabs__list"
>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item tabs__item--active"
      href="#"
    >
      <span>
        Test city
      </span>
    </a>
  </li>
</ul>
`;
